pkg/orchestrator/assetscanprocessor/malware.go

```diff
@@ -83,6 +83,7 @@ func (asp *AssetScanProcessor) reconcileResultMalwareToFindings(ctx context.Cont
 				MalwareName: item.MalwareName,
 				MalwareType: item.MalwareType,
 				Path:        item.Path,
+				RuleName:    item.RuleName,
```

Comment:
If we don't update getExistingMalwareFindingsForScan + GenerateMalwareKey, then Malware's are considered unique if MalwareName, MalwareType and Path are the same (which will be true if we restructure things to include the matches) but right now this means that if two rules match on the same file, then we through away any existing matches. Are we ok with that?

Reply:
I am adding the `ruleName` to the `MalwareKey`.